components/tools/OmeroM/src/helper/strToMapAnnotation.m

```diff
@@ -0,0 +1,100 @@
+function ma = strToMapAnnotation(session, str, varargin)
+% strToMapAnnotation returns MapAnnotation object of OMERO from
+% string array or cell array of strings
+%
+% SYNTAX
+% ma = strToMapAnnotation(session,str)
+% ma = strToMapAnnotation(session,str,iseditable)
+%
+% REQUIREMENTS
+%
+%   OMERO.matlab toolbox
+%   https://docs.openmicroscopy.org/latest/omero/developers/Matlab.html
+%
+%   Before using this function, you need to run an equivalent of the
+%   following command.
+%
+%     client = loadOmero('demo.openmicroscopy.org', 4064)
+%
+% INPUT ARGUMENTS
+% session     omero.api.ServiceFactoryPrxHelper object
+%
+% str         string array | cell array of strings
+%             Number of columns must be 2.
+%
+% iseditable  false (default) | true | 0 | 1 
+%             (Optional) If true or 1, MapAnnotation (Key-Value Pairs) will
+%             be editable via GUI (OMERO.web or OMERO.insight)
+%
+% OPTIONAL PARA<ETER/VALUE PAIRS
```

Comment:
maybe "PARAMETER"?

Reply:
See also https://github.com/ome/omero-matlab/pull/8

Reply:
Thanks for spotting.